src/utils/client.js

```diff
@@ -1,4 +1,4 @@
-import 'isomorphic-fetch';
+import 'unfetch/polyfill';
```

Comment:
To be fair, he does link to his definition of a ponyfill (I'd say others have used it to mean partial polyfill or something else, I dunno). See https://github.com/sindresorhus/ponyfill.

And actually no, we use it [here](https://github.com/simplaio/simpla/blob/2.0-preview/src/plugins/ping.js#L7) too. That's bad of me I didn't import it in the head.